cli/src/commands/contract/commands/create.ts

```diff
@@ -121,6 +124,30 @@ export default (opts: BaseCommandOptions) => {
         process.exit(1);
       }
     }
+
+    if (!options.suppressWarnings && resp.compositionWarnings.length > 0) {
+      const compositionWarningsTable = new Table({
+        head: [
+          pc.bold(pc.white('FEDERATED_GRAPH_NAME')),
+          pc.bold(pc.white('NAMESPACE')),
+          pc.bold(pc.white('FEATURE_FLAG')),
+          pc.bold(pc.white('WARNING_MESSAGE')),
+        ],
+        colWidths: [30, 30, 30, 120],
+        wordWrap: true,
+      });
+
+      console.log(pc.yellow(`We found these composition warnings, while composing the federated graph.`));
```

Comment:
`The following warnings were produced while composing the federated graph:`

Reply:
Done